examples/with-tigris/package.json

```
@@ -0,0 +1,25 @@
+{
+  "private": true,
+  "scripts": {
+    "predev": "APP_ENV=development npm run setup",
+    "dev": "next dev",
+    "build": "next build",
+    "postbuild": "APP_ENV=production npm run setup",
+    "start": "next start",
+    "clean": "rm -rf node_modules .next",
+    "setup": "npx ts-node scripts/setup.ts"
```

These have assumptions that eg `ts-node` is installed and I believe the way APP_ENV is used it won't work cross OS. can we rather add a comment about running `scripts/setup.ts` in the readme?
```suggestion
    "dev": "next dev",
    "build": "next build",
    "start": "next start",
```

---

Added a note in readme regarding custom `dev` and `build` scripts. This is demonstrating a feature that lets user bootstrap schema automatically, they can opt out of it. :)